src/Grpc.AspNetCore.Server/Internal/ReadOnlySequenceStream.cs

```diff
@@ -0,0 +1,167 @@
+﻿#region Copyright notice and license
+
+// Copyright 2019 The gRPC Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+#endregion
+
+using System;
+using System.Buffers;
+using System.IO;
+using System.Threading;
+using System.Threading.Tasks;
+
+namespace Grpc.AspNetCore.Server.Internal
+{
+    // Potentially remove in the future when https://github.com/dotnet/corefx/issues/31804 is implemented
+    internal class ReadOnlySequenceStream : Stream
+    {
+        private static readonly Task<int> TaskOfZero = Task.FromResult(0);
```

Comment:
const?

Reply:
`Task` not a primitive type so const is not allowed.